app/src/services/market_maker.ts

```diff
@@ -62,9 +62,11 @@ class MarketMakerService {
     return this.contract.removeFunding(amount)
   }
 
-  getActualPrice = async (): Promise<{ actualPriceForYes: number; actualPriceForNo: number }> => {
-    const { address } = this.contract
-    const { balanceOfForYes, balanceOfForNo } = await this.getBalanceInformation(address)
+  getActualPrice = async (balanceInformation: {
+    balanceOfForYes: BigNumber
+    balanceOfForNo: BigNumber
+  }): Promise<{ actualPriceForYes: number; actualPriceForNo: number }> => {
```

Comment:
I think this doesn't need to be async anymore? Also, it doesn't use the underlying contract, so this could either be a static method or a separate util (I think a static method is fine).

Reply:
> I think this doesn't need to be async anymore? Also, it doesn't use the underlying contract, so this could either be a static method or a separate util (I think a static method is fine).

Good catch, Done!